libs/category/src/models/get-category-response.ts

```diff
@@ -13,7 +13,16 @@ export interface DaffGetCategoryResponse<
 	V extends DaffGenericCategory<V> = DaffCategory,
 	W extends DaffProduct = DaffProduct
 > {
+	/**
+	 * The list of products described by the requested category.
```

Comment:
I would say `The list of products that belong to the requested category`

Reply:
changed